pt/python_introduction/README.md

```diff
@@ -212,8 +212,8 @@ Mas, e se digitarmos o nome errado? Você consegue adivinhar o que aconteceria?
 
```

Comment:
Na linha 129: retirar barra de escape `\*` em **operadores**

Reply:
But she didn't add that...